<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>configwidgets</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ConfigCheckBox</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ConfigCheckBox" name="checkBox">
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ConfigLineEdit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ConfigLineEdit" name="lineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ConfigSpinBox</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ConfigSpinBox" name="spinBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>ConfigDoubleSpinBox</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ConfigDoubleSpinBox" name="doubleSpinBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="configDateEditLabel">
       <property name="text">
        <string>ConfigDateEdit</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="ConfigDateEdit" name="dateEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="configTimeEditLabel">
       <property name="text">
        <string>ConfigTimeEdit</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="ConfigTimeEdit" name="timeEdit"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="configDateTimeEditLabel">
       <property name="text">
        <string>ConfigDateTimeEdit</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="ConfigDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="ConfigRadioButton" name="radioButton_2">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ConfigRadioButton1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>ConfigRadioButton2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ConfigRadioButton" name="radioButton_1">
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ConfigComboBox" name="comboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ConfigComboBox</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>107</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_show">
       <property name="text">
        <string>show Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string>reset Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_restart">
       <property name="text">
        <string>restart</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">configwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">configwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">configwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">configwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">configwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>configwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigLineEdit</class>
   <extends>QLineEdit</extends>
   <header>configwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>configwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>ConfigRadioButton</class>
   <extends>QRadioButton</extends>
   <header>configwidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
